### ISSUE TITLE
Google's web service is not working

### ISSUE THREAD

**Issue by Waltorvi** [Bionus/imgbrd-grabber#3212]
By default Google web service redirects to the wrong link, which causes the usual google search engine to open, to fix this you need to change the link to https://lens.google.com/uploadbyurl?url={url}. Checked on Edge browser

**Comment by Bionus** [comment]
Indeed, seems like they don't allow the old URL for a while. That's unfortunate, their Google Lens thing is not great.
Confirmed in https://stackoverflow.com/questions/8325055/how-to-compose-the-url-for-a-reverse-google-image-search
